Stripe/STPAddressFieldTableViewCell.m

```diff
@@ -262,7 +262,7 @@ - (void)nextTapped:(__unused id)sender {
 #pragma mark - STPFormTextFieldDelegate
 
 - (void)formTextFieldTextDidChange:(STPFormTextField *)textField {
-    if (self.type != STPAddressFieldTypeCountry) {
+    if (self.type != STPAddressFieldTypeCountry && self.type != STPAddressFieldTypeSEPACountry) {
```

Comment:
This fixes a bug I caught while adding tests – for SEPA, we were using the country's display name (i.e. "Germany") in the SourceParams object rather than the ISO code ("DE").